config.template.py

```diff
@@ -18,7 +19,7 @@ class Config(object):
     # Unique prefix that defines VM name space for this Tango
     # version. When working in development, this prefix should be your
     # unique identifier. The "prod" prefix is reserved for production
-    PREFIX = "local"
+    PREFIX = os.getenv("DOCKER_DEPLOYMENT", "local").lower()
```

Comment:
It would be easier for users to modify this line of config if the "local" part is separated out, such that it's the only thing on the RHS (similar to setting DOCKER_VOLUME_PATH)